src/quo2/components/banners/banner/view.cljs

```diff
@@ -3,22 +3,29 @@
             [quo2.components.counter.counter :as counter]
             [quo2.components.icon :as icons]
             [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
             [react-native.core :as rn]))
 
 (defn banner
   [{:keys [show-pin? latest-pin-text pins-count on-press]}]
-  [rn/touchable-opacity
-   {:accessibility-label :pinned-banner
-    :style               style/container
-    :active-opacity      1
-    :on-press            on-press}
-   (when show-pin? [icons/icon :i/pin {:size 20}])
-   [text/text
-    {:number-of-lines 1
-     :size            :paragraph-2
-     :style           {:margin-left 10 :margin-right 50}}
-    latest-pin-text]
-   [rn/view
-    {:accessibility-label :pins-count
-     :style               style/counter}
-    (when (pos? pins-count) [counter/counter {:type :secondary} pins-count])]])
\ No newline at end of file
+  (when (pos? pins-count)
+    [rn/touchable-opacity
+     {:accessibility-label :pinned-banner
+      :style               style/container
+      :active-opacity      1
+      :on-press            on-press}
+     (when show-pin?
+       [icons/icon :i/pin
+        {:color (colors/theme-colors
+                 colors/neutral-100
+                 colors/white)
+         :size  20}])
+     [text/text
+      {:number-of-lines 1
+       :size            :paragraph-2
+       :style           {:margin-left 10 :margin-right 50}}
```

Comment:
The style can be extract to `style.cljs`.

Reply:
sure 👍